src/components/MarketList/index.js

```diff
@@ -231,21 +230,23 @@ class MarketList extends Component {
             </div>
           </div>
         </div>
-        <div className="marketListPage__controls">
-          <div className="container">
-            <div className="row">
-              <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
-                <InteractionButton
-                  onClick={this.handleCreateMarket}
-                  className="marketStats__control btn btn-default"
-                  whitelistRequired
-                >
-                  Create Market
-                </InteractionButton>
+        {process.env.WHITELIST[defaultAccount] && (
+          <div className="marketListPage__controls">
+            <div className="container">
+              <div className="row">
+                <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
```

Comment:
I think this should be ```col-xs-10 col-xs-offset-2 col-sm-12``` as far as I know the grid layout we are using is divided in 12 columns, right?